libs/data/src/lib/json/mobs.json

```diff
@@ -75756,28 +75756,28 @@
     "fr": ""
   },
   "12627": {
-    "en": "morbol",
```

Comment:
Did you make these changes by hand? because they're extracted from the game so these changes will be overriden next time the extractors are going to be used, which will be sooner than later for sure.

Reply:
Yes, I do it manually, I haven't check if it was extracted.

It's strange , the first letter is not capitalized, seeing this mob in game and it was. Maybe we can try to re-run the extractor ?

However, we can keep this change and it would be overwritted?

Reply:
Yeah it'll just be overwritten, this is due to some small capitalization issue, probably not extracting from the right column but TC should handle this fine iirc.